src/System Application/App/Translation/src/Translation.Codeunit.al

```diff
@@ -195,4 +195,16 @@ codeunit 3711 Translation
     begin
         TranslationImplementation.ShowForAllRecords(TableId, FieldId);
     end;
-}
+
+    /// <summary>
+    /// Gets all available translations for a specific record field as a temporary buffer
+    /// </summary>
+    /// <param name="RecVariant">Source record variant</param>
+    /// <param name="FieldId">Field ID to get translations for</param>
```

Comment:
We should document that the all translations will be returned if the FieldId is 0.